static_home_page/templates/base.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+{% load static %}
+<head>
+	<meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<title>FitMeet</title>
+		<link rel="stylesheet" href="{% static "web_page_css.css" %}">
```

Comment:
```suggestion
		{% block extrahead %}{% endblock %}
```
This line will allow to each block inject its own style sheet. 
On each sub-template, we need to specify the following blocks:
{% extends "base.html" %}

{% block extrahead %}
    <link rel="stylesheet" type="text/css" href="{% static 'css/child.css' %}">
{% endblock %}

{% block content %}
    the content of the template
{% endblock %}

Reply:
In your suggestion, it seems like you want to replace line 9 with "{% block extrahead %}
{% endblock %}". However, doing so would remove the CSS from the base.html file. Did you perhaps mean to add this line somewhere else?

Reply:
The whole Idea is that we all agreed with to use the same style sheet as in the index.html for the whole project In our meeting on Sunday that you missed btw so maybe we forgot to mention . if this style does not match part of a page you are working on sure you can use this {% block extrahead %}{% endblock %} to inject another style.

Reply:
I Thought about your suggestion, but I think it is better to leave it as it is because IMHO
Having a separate CSS file for each web page can make it easier to manage and maintain the styles specific to that page. It allows for clear separation of concerns, making it easier to understand and update the styles for a particular page without affecting others.

However, if our website is relatively small and consists of just a few pages, combining the CSS into a single file might be more efficient.
Combining multiple CSS files into a single file reduces the number of separate requests needed for CSS resources. Instead of making individual requests for each CSS file, the browser can fetch and load a single CSS file, resulting in fewer round trips to the server. This consolidation reduces the overhead associated with establishing and managing multiple network connections, resulting in faster page load times.

Reply:
That is another approach, but everybody should be cautious when applying new styles. it can cause many bugs by overriding styles.